routes/attempt_answers-api.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * All routes for Attempts Data are defined here
+ *
+ * NOTE: There is no "EDIT" route here as editing an attempt itself
+ * doesn't make sense.  A user might edit a attempt_answer (e.g. if they
+ * change their mind), but the attempt itself should never be changed.
+
+ * Since this file is loaded in server.js into api/quizzes,
+ *   these routes are mounted onto /api/quizzes
+ * See: https://expressjs.com/en/guide/using-middleware.html#middleware.router
+ */
+
+const express = require("express");
+const router = express.Router();
+const {
+  createAttemptAnswer,
+  getAttemptAnswer,
+  deleteAttemptAnswer,
+} = require("../db/queries");
+
+const hardcoded_attempt_id = 2;
+
+router.post("/", (req, res) => {
+  const { attempt_id, question_id, answer_id } = req.body;
+
+  // Set some default data
+  let queryParams = [hardcoded_attempt_id, 5, 25];
+
+  if (attempt_id && question_id && answer_id) {
+    // Real data was sent so replace the fake data with the actual
+    queryParams = [attempt_id, question_id, answer_id];
+  }
+
+  createAttemptAnswer(queryParams)
+    .then((data) => {
+      console.log(
+        "🚀 ~ file: attempt_answers-api.js:44 ~ router.post ~ data:",
+        data
+      );
+      const attempts = data;
+      res.json({ attempts });
+    })
+    .catch((err) => {
+      res.status(500).json({ error: err.message });
+    });
+});
+
+router.get("/:id", (req, res) => {
+  const { user_id } = req.body;
```

Comment:
Just a note for future: I think we can get this in the req.session

Reply:
Yeah, right now it's just a placeholder; once we figure out where the `user_id` will *actually* be coming from, I'm going to go back and update all of these to get the user_id from the correct place, whatever that may be.